pkg/toolset/jwt.go

```diff
@@ -19,33 +19,31 @@ import (
 
 func generateJWTApiToken(nodeConfig *configuration.Configuration, args []string) error {
 
-	fs := flag.NewFlagSet("", flag.ExitOnError)
-	outputJSON := fs.Bool("json", false, "format output as JSON")
-	databasePath := fs.String("database", "", "the path to the p2p database folder (optional)")
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+	databasePathFlag := fs.String(FlagToolDatabasePath, "", "the path to the p2p database folder (optional)")
+	outputJSONFlag := fs.Bool(FlagToolOutputJSON, false, FlagToolDescriptionOutputJSON)
 
 	fs.Usage = func() {
 		fmt.Fprintf(os.Stderr, "Usage of %s:\n", ToolJWTApi)
 		fs.PrintDefaults()
+		println(fmt.Sprintf("\nexample: %s --%s %s",
+			ToolJWTApi,
+			FlagToolDatabasePath,
+			"p2pstore"))
 	}
 
-	if err := fs.Parse(args); err != nil {
+	if err := parseFlagSet(fs, args); err != nil {
 		return err
 	}
 
-	// Check if all parameters were parsed
-	if fs.NArg() != 0 {
-		fs.Usage()
-		os.Exit(2)
-	}
-
 	salt := nodeConfig.String(restapi.CfgRestAPIJWTAuthSalt)
```

Comment:
do we know if this work with `--config`? Else better remove the config.json dependance altogether and let the users pass the required values as flags

Reply:
I removed it!